lib/pages/farm/usecases/get_farm_data_use_case.dart

```diff
@@ -7,23 +7,31 @@ class GetFarmDataUseCase {
   final GysrApi gysrApi = GysrApi();
 
   Future<List<FarmModel>> fetchAllFarms(String owner) async {
-    final response = await gysrApi.fetchAllFarms(owner);
-    if (response.hasException) throw response.exception.toString();
-    return _mapQueryResultToFarmModel(
-      response.data!['pools'] as List<dynamic>,
-      false,
-    );
+    try {
+      final response = await gysrApi.fetchAllFarms(owner);
+      if (response.hasException) throw response.exception.toString();
+      return _mapQueryResultToFarmModel(
+        response.data!['pools'] as List<dynamic>,
+        false,
+      );
+    } catch (e) {
```

Comment:
This is sub-optimal error handling, but till we're not revamping it, it's better than nothing.